tests/packagedcode/data/plugin/podspec-package-expected.json

```diff
@@ -0,0 +1,74 @@
+{
+  "headers": [
+    {
+      "tool_name": "scancode-toolkit",
+      "options": {
+        "input": "<path>",
+        "--json": "<file>",
+        "--package": true,
+        "--processes": "-1",
+        "--strip-root": true
+      },
+      "notice": "Generated with ScanCode and provided on an \"AS IS\" BASIS, WITHOUT WARRANTIES\nOR CONDITIONS OF ANY KIND, either express or implied. No content created from\nScanCode should be considered or used as legal advice. Consult an Attorney\nfor any legal advice.\nScanCode is a free software code scanning tool from nexB Inc. and others.\nVisit https://github.com/nexB/scancode-toolkit/ for support and download.",
+      "message": null,
+      "errors": [],
+      "extra_data": {
+        "files_count": 1
+      }
+    }
+  ],
+  "files": [
+    {
+      "path": "SwiftLib.podspec",
+      "type": "file",
+      "packages": [
+        {
+          "type": "pods",
+          "namespace": null,
+          "name": "SwiftLib",
+          "version": "0.0.1",
+          "qualifiers": {},
+          "subpath": null,
+          "primary_language": "Objective-C",
+          "description": " This CocoaPods library helps you perform calculation.",
+          "release_date": null,
+          "parties": [
+            {
+              "type": "person",
+              "role": "author",
+              "name": "alizainprasla",
+              "email": "alizainprasla@gmail.com",
+              "url": null
+            }
+          ],
+          "keywords": [],
+          "homepage_url": "https://github.com/alizainprasla/swiftlib",
+          "download_url": null,
+          "size": null,
+          "sha1": null,
+          "md5": null,
+          "sha256": null,
+          "sha512": null,
+          "bug_tracking_url": null,
+          "code_view_url": null,
+          "vcs_url": ":git => https://github.com/alizainprasla/swiftlib.git, :tag =>",
```

Comment:
These need to be valid urls

Reply:
Please see https://github.com/nexB/scancode-toolkit/pull/2067#discussion_r445453088
I have made changes